src/libraries/System.Reflection.Extensions/README.md

```diff
@@ -0,0 +1,7 @@
+# System.Reflection.Extensions
+This supports forwarding types to the runtime including [CustomAttributeExtensions](https://learn.microsoft.com/dotnet/api/system.reflection.CustomAttributeExtensions), [InterfaceMapping](https://learn.microsoft.com/dotnet/api/system.reflection.InterfaceMapping) and [RuntimeReflectionExtensions](https://learn.microsoft.com/dotnet/api/system.reflection.RuntimeReflectionExtensions).
+
+## Status: [Legacy](../../libraries/README.md#development-statuses)
+
+## Deployment
+[System.Reflection.Extensions](https://www.nuget.org/packages/System.Reflection.Extensions) is included in the shared framework.
```

Comment:
@steveharter / @teo-tsirpanis is this content suggestion accurate?

```suggestion
## Status: [Legacy](../../libraries/README.md#development-statuses)
This package was used in the past to expose common functionality across different runtimes in a uniform way. It continues to ship as part of the shared framework, but it is frozen for compatibility.

## Deployment
[System.Reflection.Extensions](https://www.nuget.org/packages/System.Reflection.Extensions) is included in the shared framework. The package does not need to be installed into any project compatible with .NET Standard 2.0.
```

Reply:
I kept it since it is useful here IMO.